host/src/FunctionsNetHost/AppLoader/AppLoader.cs

```diff
@@ -14,37 +20,35 @@ internal sealed class AppLoader : IDisposable
         private IntPtr _hostContextHandle = IntPtr.Zero;
         private bool _disposed;
 
-        internal AppLoader()
-        {
-            LoadHostfxrLibrary();
-        }
-
-        private void LoadHostfxrLibrary()
-        {
-            // If having problems with the managed host, enable the following:
-            // Environment.SetEnvironmentVariable("COREHOST_TRACE", "1");
-            // In Unix environment, you need to run the below command in the terminal to set the environment variable.
-            // export COREHOST_TRACE=1
-
-            var hostfxrFullPath = NetHost.GetHostFxrPath();
-            Logger.LogTrace($"hostfxr path:{hostfxrFullPath}");
-
-            _hostfxrHandle = NativeLibrary.Load(hostfxrFullPath);
-            if (_hostfxrHandle == IntPtr.Zero)
-            {
-                Logger.Log($"Failed to load hostfxr. hostfxr path:{hostfxrFullPath}");
-                return;
-            }
-
-            Logger.LogTrace($"hostfxr library loaded successfully.");
-        }
-
         internal int RunApplication(string? assemblyPath)
         {
             ArgumentNullException.ThrowIfNull(assemblyPath, nameof(assemblyPath));
 
             unsafe
             {
+                var parameters = new NetHost.get_hostfxr_parameters
+                {
+                    size = sizeof(NetHost.get_hostfxr_parameters),
+                    assembly_path = GetCharArrayPointer(assemblyPath)
+                };
+
+                var sw = Stopwatch.StartNew();
```

Comment:
The StopWatch instrumentation can be removed later once we get a sense of how much time this takes in private stamps.

Reply:
These are removed.